src/UCTSearch.h

```diff
@@ -84,6 +84,8 @@ class UCTSearch {
     SearchResult play_simulation(BoardHistory& bh, UCTNode* const node);
 
 private:
+    std::string last_pv;
+    int last_depth;
```

Comment:
Please move these down to be with all the other member variables, and follow the m_member_var naming convention (a few others have crept in, we can fix those later but let's not add more).

Reply:
Thanks, done